repository/kafka/src/main/java/com/expediagroup/streamplatform/streamregistry/repository/kafka/KafkaConfiguration.java

```diff
@@ -37,40 +37,43 @@ EventCorrelator eventCorrelator() {
 
   @Bean
   EventSender eventSender(
-      @Value("${repository.kafka.bootstrapServers}") String bootstrapServers,
```

Comment:
Can we avoid the formatting flip-flip :)

Reply:
Should we be using 2 or 4 space indentation?

I'm happy to change back but I think this indentation is now consistent with the formatting in other classes.

Reply:
We could do a separate pr just to fix formatting if it affects many classes? Perhaps adding .editorconfig at same time.